src/components/Tree/Tree.stories.tsx

```diff
@@ -54,84 +21,30 @@ export default {
 } as Meta<TreeProps>;
 
 export const Tree: Story<TreeProps> = (args: TreeProps) => {
-    const [nodesState, setNodes] = useState(listToTree(apiNodes));
-
-    const updateItemsPosition = (
-        nodes: TreeListItem[], // nodes without the moved node
-        targetParentId: string | null,
-        movedItemTargetIndex: number,
-        movedItem: TreeListItem,
-    ): TreeListItem[] => {
-        nodes.splice(movedItemTargetIndex, 0, movedItem);
-        const sortedSameLevelItems = nodes.filter((node) => node.parentId === targetParentId);
-
-        const modifiedSameLevelArray: TreeListItem[] = [];
-        const changedItemsIds: string[] = [];
-
-        sortedSameLevelItems.forEach((item, index) => {
-            const previousItem = modifiedSameLevelArray[index - 1];
-            let modifiedItem = { ...item };
-
-            const shouldUpdate =
-                (index === 0 && item.sort === null) || // First item without a sort value
-                (index < movedItemTargetIndex && item.sort === null) || // Previous item without a sort value
-                index === movedItemTargetIndex || // Moved item
-                (index > movedItemTargetIndex &&
-                    item.sort !== null &&
-                    previousItem &&
-                    previousItem.sort &&
-                    item.sort < previousItem.sort); // Following item with a sort prop lower than on previousItem
-
-            if (shouldUpdate) {
-                const incrementedPreviousItemSort = previousItem && previousItem.sort ? previousItem.sort + 1 : null;
-                const updatedSortValue = index === 0 ? 0 : incrementedPreviousItemSort;
+    const [nodesState, setNodes] = useState(apiNodes);
```

Comment:
rename nodesState to nodes or setNodes to SetNodesState. Getter/Setter should have same name.

Reply:
Done